README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/pokutuna/p5-JSON-Schema-Validator.png?branch=master)](https://travis-ci.org/pokutuna/p5-JSON-Schema-Validator)
```

Comment:
You can choose SVG version of the badge, we recommend it rather than PNG version.

e.g.: `+[![Build Status](https://travis-ci.org/pokutuna/p5-JSON-Schema-Validator.svg?branch=master`